projects/plugins/crm/includes/ZeroBSCRM.InvoiceBuilder.php

```diff
@@ -1096,59 +1096,64 @@ function zeroBSCRM_invoicing_generateInvoiceHTML( $invoice_id = -1, $template =
 	// == Partials (Transactions against Invs)
 	$partials_table = '';
 
-	if ( $invoice['total'] == 0 ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
-		$partials_table .= '<table id="partials" class="hide table-totals striped">';
-	} else {
-		$partials_table .= '<table id="partials" class="table-totals striped">';
-	}
+	// Check if partial payments are disabled
+	if ( ! isset( $invsettings['invoicing_disable_partial_payments'] ) || $invsettings['invoicing_disable_partial_payments'] !== '1' ) {
```

Comment:
I believe this is an int rather than a string (but do verify).

```suggestion
	if ( ! isset( $invsettings['invoicing_disable_partial_payments'] ) || $invsettings['invoicing_disable_partial_payments'] !== 1 ) {
```

Reply:
e32c09fa774